Library/Homebrew/service.rb

```diff
@@ -355,6 +355,16 @@ def command
       @run&.map(&:to_s)
     end
 
+    sig { params(value: Integer).returns(T.nilable(Integer)) }
+    def nice(value)
```

Comment:
```suggestion
    def nice(value = nil)
```

Sorry, this bit is still needed and then then `when nil` below will also be needed. 

We want users to be able to call `nice()` or `nice(5)` but not `nice("5")` or `nice(nil`).

Does that make sense?

Sorry for all the back and forth!

Reply:
When would be want to call `nice` without arguments though?

Reply:
I don't really see a reason why any of these methods would need to behave like a getter.